easybuild/easyconfigs/r/Raptor/Raptor-2.0.16-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+# Based on Raptor2 RPM spec file:
+# https://git.rockylinux.org/staging/rpms/raptor2/-/blob/r8/SPECS/raptor2.spec
+# The RPM patches have been integrated in the version 2.0.16
+##
+easyblock = 'ConfigureMake'
+
+name = 'Raptor'
+version = '2.0.16'
+homepage = 'https://librdf.org/raptor/'
+description = """Set of parsers and serializers that generate Resource Description Framework
+(RDF) triples by parsing syntaxes or serialize the triples into a syntax."""
+# software_license = 'LicenseLGPLv2.1 + LicenseGPLv2 + LicenseApachev2'
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://download.librdf.org/source']
+sources = ['raptor2-%(version)s.tar.gz']
+checksums = ['089db78d7ac982354bdbf39d973baf09581e6904ac4c92a98c5caadb3de44680']
+
+builddependencies = [
+    ('Autotools',  '20210128'),
+    ('make',       '4.3'),
+    ('binutils',   '2.36.1'),
+    ('libxml2',    '2.9.10'),
+    ('libxslt',    '1.1.34'),
+    ('cURL',       '7.76.0'),
+    ('ICU',        '69.1'),
+]
```

Comment:
Some of these should be `dependencies` and not `builddependencies`.

Reply:
Done